rules-reviewed/eap6/websphere/websphere-xml.windup.xml

```diff
@@ -6,11 +6,11 @@
             This ruleset provides analysis of WebSphere proprietary xml files and constructs that may require individual attention when migrating to JBoss EAP 6+.
         </description>
         <dependencies>
-            <addon id="org.jboss.windup.rules,windup-rules-xml,2.3.0.Final"/>
+            <addon id="org.jboss.windup.rules,windup-rules-xml,2.7.0.Final"/>
         </dependencies>
         <sourceTechnology id="websphere"/>
-        <targetTechnology id="eap" versionRange="[6,7)"/>
-        <tag>reviewed-2015-06-17</tag>
+        <targetTechnology id="eap" versionRange="[6,7]"/>
```

Comment:
I would keep this rather in one ruleset file as the suggestion for both are the same and I included specific links to 6 or 7 for reference as user can use what they needed for his concrete migration. This would reduce the maintenance later I think.